src/wazuh_modules/inventory_harvester/tests/unit/systemInventoryUpsertElement_test.cpp

```diff
@@ -176,17 +192,6 @@ TEST_F(SystemInventoryUpsertElement, validAgentID_Ports)
         R"({"id":"001_portItemId","operation":"INSERTED","data":{"agent":{"id":"001","name":"agentName","ip":"agentIp","version":"agentVersion"},"destination":{"ip":"portRemoteIp","port":1234},"file":{"inode":"1111"},"host":{"network":{"egress":{"queue":7000},"ingress":{"queue":11000}}},"interface":{"state":"portState"},"network":{"transport":"portProtocol"},"process":{"name":"portProcess","pid":4321},"source":{"ip":"portLocalIp","port":7777}}})");
 }
 
```

Comment:
Why did you remove the `emptyAgentID_Ports` test?

Reply:
Restore thanks!